include/dxc/Support/SPIRVOptions.h

```diff
@@ -1,24 +1,25 @@
-//===-- EmitSPIRVOptions.h - Options for SPIR-V CodeGen ---------*- C++ -*-===//
-//
-//                     The LLVM Compiler Infrastructure
-//
-// This file is distributed under the University of Illinois Open Source
-// License. See LICENSE.TXT for details.
-//
-//===----------------------------------------------------------------------===//
-#ifndef LLVM_CLANG_SPIRV_EMITSPIRVOPTIONS_H
-#define LLVM_CLANG_SPIRV_EMITSPIRVOPTIONS_H
+///////////////////////////////////////////////////////////////////////////////
+//                                                                           //
+// SPIRVOptions.h                                                            //
+// Copyright (C) Microsoft Corporation. All rights reserved.                 //
+// This file is distributed under the University of Illinois Open Source     //
+// License. See LICENSE.TXT for details.                                     //
+//                                                                           //
+//                                                                           //
+///////////////////////////////////////////////////////////////////////////////
 
-#include <string>
-#include <vector>
+#pragma once
 
-#include "llvm/ADT/SmallVector.h"
+#ifndef LLVM_SPIRV_OPTIONS_H
+#define LLVM_SPIRV_OPTIONS_H
+
+#ifdef ENABLE_SPIRV_CODEGEN
+
+#include "llvm/ADT/ArrayRef.h"
 #include "llvm/ADT/StringRef.h"
+#include "llvm/Option/ArgList.h"
 
-namespace clang {
-namespace spirv {
-/// Memory layout rules
-enum class LayoutRule {
+enum class SpirvLayoutRule {
```

Comment:
We probably need namespaces for this header.

Reply:
Done